### PR TITLE
revise organization

### PR DESCRIPTION
**Pull request by dragonylffly**
Signed-off-by: Li Wang <li.wang@kylin-cloud.com>

### CODE REVIEW

**Comment by dachary** [comment]
@dragonylffly did the organization name change ? Or is it a new organization but the previous one remains ?

**Comment by dragonylffly** [comment]
> @dragonylffly did the organization name change ? Or is it a new organization but the previous one remains ?

The former, thanks

File: .organizationmap [hunk 1]
```diff
@@ -323,10 +323,11 @@ The Linux Box <contact@linuxbox.com> Ali Maredia <ali@linuxbox.com>
 The Linux Box <contact@linuxbox.com> Casey Bodley <casey@linuxbox.com>
 The Linux Box <contact@linuxbox.com> Matt Benjamin <matt@linuxbox.com>
 The University of Arizona <contact@arizona.edu> James Ryan Cresawn <jrcresawn@gmail.com>
-Ubuntu Kylin <contact@ubuntukylin.com> Li Wang <liwang@ubuntukylin.com>
```
Comment (dachary) [inline .organizationmap]: Kylin Cloud &lt;contact@kylin-cloud.com> &lt;liwang@ubuntukylin.com>

File: .organizationmap [hunk 2]
```diff
@@ -323,10 +323,11 @@ The Linux Box <contact@linuxbox.com> Ali Maredia <ali@linuxbox.com>
...
 Ubuntu Kylin <contact@ubuntukylin.com> Min Chen <minchen@ubuntukylin.com>
```
Comment (dachary) [inline .organizationmap]: Kylin Cloud &lt;contact@kylin-cloud.com> &lt;minchen@ubuntukylin.com>

File: .organizationmap [hunk 3]
```diff
@@ -323,10 +323,11 @@ The Linux Box <contact@linuxbox.com> Ali Maredia <ali@linuxbox.com>
...
-Ubuntu Kylin <contact@ubuntukylin.com> MingXin Liu <mingxinliu@ubuntukylin.com>
```
Comment (dachary) [inline .organizationmap]: Kylin Cloud &lt;contact@kylin-cloud.com> &lt;mingxinliu@ubuntukylin.com>

**Pull request merged**